### DISCUSSION TITLE
cannot import name 'set_data_path' from 'spacy.util'

### DISCUSSION THREAD

**Discussion by erotavlas** [explosion/spaCy#7758] [Help: Coding & Implementations] [answered]
Do we still need to explicitly set the data path in spacy 3.0?
I previously had this working in spacy 2.x

```
from spacy.util import set_data_path
    dataPath = Path("data")
    set_data_path(dataPath)
```
But now I'm getting this error

`ImportError: cannot import name 'set_data_path' from 'spacy.util' (D:\Anaconda\envs\spacy3.0\lib\site-packages\spacy\util.py)`

**Comment by polm** [comment, marked as answer]
That function is gone, [symlinks are deprecated](https://spacy.io/usage/v3#incompat-removed:~:text=not%20needed%2C%20symlinks%20are%20deprecated) in v3. 

In v3 you use the registry and/or entry points to get models, so they work more like ordinary Python packages.
